### DISCUSSION TITLE
How to set up a UE5 (Unreal Engine) repo?

### DISCUSSION THREAD

**Discussion by SpainVstudio** [community/community#130989] [Enterprise]
I need to create a repo in git but the project size is 28gb and i cant push or anything please help

PD: I have GitHub Enterprise
![image](https://github.com/community/community/assets/174537079/8a0b6879-8c4a-4591-9b73-1110990a76eb)

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @SpainVstudio!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the Enterprise category. I've gone ahead and moved it for you. Good luck!
